src/components/Causes/Causes.tsx

```diff
@@ -1,80 +1,146 @@
 import { Typography } from "@mui/material";
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import { textMargin } from "./Causes.styles";
-
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import { orderedListItemStyle, textMargin } from "./Causes.styles";
 
+const FootNote = ({ url, linkText }: { url: string; linkText: string }) => {
+  return (
+    <li style={orderedListItemStyle}>
+      <a href={url} rel="noopener noreferrer" target="_blank">
+        {linkText}
+      </a>
+    </li>
+  );
+};
 
 const Causes = () => {
+  // Editors note: these counts will be off in dev mode by a factor of 2 b/c in dev mode
+  // react calls component function bodies twice to check for side effects during development.
+  // This is fine in a production build
+  let superCount = 1;
+  const Superscript = () => {
+    return <sup>{superCount++}</sup>;
+  };
```

Comment:
I'm a bit wary of this code, and think there may be a better way to do it with React state somehow. Still mulling on a solution, though.